docs/api.asciidoc

```diff
@@ -75,18 +80,22 @@ Arguments:
 
   * `name`: A name for your transaction. Transactions are grouped by name. **Required**.
   * `type`: A `type` for your transaction eg. `db.postgresql.sql`.
-  * `rack_env: env`: An optional Rack `env` used to enrich the transaction with information about the current request.
-  * `&block`: An optional block to wrap with the transaction. The block is passed the transaction as an optional argument.
+  * `context:`: An optional Context used to enrich the transaction with
+  information about the current request.
+  * `&block`: An optional block to wrap with the transaction. The block is passed the
+  transaction as an optional argument.
```

Comment:
The formulation `to wrap with the transaction`  is confusing me a bit. I think simply saying `An optional block that gets passed to the transaction` should be sufficient.